spec/parity/backup_spec.rb

```diff
@@ -23,27 +23,36 @@
   end
 
   it "restores backups to staging from production" do
+    stub_heroku_app_name
     allow(Kernel).to receive(:system)
 
     Parity::Backup.new(from: "production", to: "staging").restore
 
     expect(Kernel).
       to have_received(:system).
+      with(heroku_staging_pg_reset)
+    expect(Kernel).
+      to have_received(:system).
       with(heroku_production_to_staging_passthrough)
   end
 
   it "restores backups to staging from development" do
+    stub_heroku_app_name
     allow(IO).to receive(:read).and_return(database_fixture)
     allow(Kernel).to receive(:system)
 
     Parity::Backup.new(from: "development", to: "staging").restore
 
     expect(Kernel).
       to have_received(:system).
+      with(heroku_staging_pg_reset)
```

Comment:
I believe this test requires that `parity-staging` is the same in both `stub_heroku_app_name` and `heroku_staging_pg_reset`. That wasn't clear to me in reading the spec initially.

Consider updating both methods to taking name as an optional argument so that this can become:

```
app_name = "parity-staging"
stub_heroku_app_name(app_name)

.with(heroku_staging_pg_reset(app_name))
```